.github/workflows/eden.yml

```diff
@@ -1,6 +1,9 @@
 ---
-name: Eden
-on:  # yamllint disable-line rule:truthy
+name: lf-edge/eden Test suite
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
```

Comment:
Will dispatch work if we take into account branches below? I can see:

- `if: github.event.review.state == 'approved'`
- `if: github.ref == 'refs/heads/master'`
- `if: startsWith(github.ref, 'refs/tags')`

Do we expect any of them to be true during `workflow_dispatch`?

Reply:
I am facing weird issue where the GHA are completed within sec.  https://github.com/yash-zededa/eve/actions/runs/6453036247 

The above actions should have initiated from the start but it got completed in like 8 sec not sure what is wrong. Trying to investigate